src/packages/api-playground/api-playground.ts

```diff
@@ -5,5 +5,56 @@ import { getEnv } from '../../env.js';
 export default (app: Express) => {
   const { GQL_API_URL: endpointURL, GQL_API_WS_URL: subscriptionsEndpoint } = getEnv();
   const initialQuery = '{ plan { id name } }';
-  app.use('/api-playground', altairExpress({ endpointURL, initialQuery, subscriptionsEndpoint }));
+  const initialPreRequestScript =
+  `
+  const nowInSeconds = () => Date.now() / 1000;
```

Comment:
Would it be possible to move this script into it's own file so it's easier to maintain? We could load it as a string when the outer function runs.

Reply:
Thanks for the suggestion! I've moved it to its own file.